experiment/dispatcher.py

```diff
@@ -92,7 +92,11 @@ def dispatcher_main():
     # Set this here because we get failures if we do it in measurer for some
     # reason.
     multiprocessing.set_start_method('spawn')
-    builder.gcb_build_base_images()
+    db_utils.initialize()
+    if os.getenv('LOCAL_EXPERIMENT'):
```

Comment:
shouldn't this if block be inside db_utils.initialize.

Reply:
I don't think so. `initialize` is called by any process using the db. This code is meant to be called only once.